src/components/Banner/banner.css

```diff
@@ -0,0 +1,36 @@
+.banner {
+  width: 90vw;
+  height: 70vh;
+  background-repeat: no-repeat;
+  background-size: contain;
+  margin: 2rem auto;
+  border-radius: 10px;
+  position: relative;
+}
+
+.banner img {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+  opacity: 0.1;
+  border-radius: 10px;
+}
+
+.banner-cover {
+  background: linear-gradient(to right, transparent, var(--primary));
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  border-radius: 10px;
+}
+
+.banner-heading {
+  position: absolute;
+  top: 50%;
```

Comment:
Suggestion: Try to avoid using large values in the top or margin it will affect in responsiveness

Reply:
I used percentage for this exact reason so that no matter the screen size, It will always work